nbgrader/docs/source/contributor_guide/installation_developer.rst

```diff
@@ -29,16 +35,17 @@ whenever you update the nbgrader repository.
 The serverextension is a Python module inside nbgrader, so only an
 enable step is needed.
 To install and enable all the frontend nbextensions (*assignment list*,
-*create assignment*, and *formgrader*) along with the server extensions
-(*assignment list* and *formgrader*) run::
+*create assignment*, and *formgrader*) and the frontend labextensions
+along with the server extensions (*assignment list* and *formgrader*) run::
 
+    jupyter labextension develop --overwrite .
```

Comment:
This command will fail unless `pip install` has been run first, as I guess the pip command builds the lab extension. Perhaps it's worth stating this explicitly, to avoid confusion?

Reply:
fixed in https://github.com/jupyter/nbgrader/pull/1588/commits/ca01cd66c7664fcf254cccbd3c67e5cdac3f466d